<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
  background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #ff9a9e, stop:0.25 #fad0c4, stop:0.5 #fbc2eb, stop:0.75 #a6c1ee, stop:1 #f6d365);
}

QWidget#centralwidget {
  background-color: rgba(255, 255, 255, 0.9);
  border-radius: 15px;
  margin: 10px 20px 20px 20px;
}

QPushButton {
  background-color: #ff6f61;
  color: white;
  font-size: 18px;
  border-radius: 10px;
  border: none;
}

QPushButton:hover {
  background-color: #ff3b2f;
}

QPushButton#button_plus, QPushButton#button_minus, QPushButton#button_multiply, QPushButton#button_clear {
  background-color: #6b5b95;
}

QPushButton#button_plus:hover, QPushButton#button_minus:hover, QPushButton#button_multiply:hover, QPushButton#button_clear:hover {
  background-color: #4a3c6e;
}

QPushButton#button_equal {
  background-color: #88d8b0;
}

QPushButton#button_equal:hover {
  background-color: #66c89f;
}

QLineEdit {
  background-color: #333;
  color: white;
  font-size: 28px;
  padding: 10px;
  border-radius: 5px;
}

QLCDNumber {
  background-color: #333;
  color: white;
  font-size: 100px;
}

QLabel {
  color: #fff;
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_text">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>351</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_clear">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0)</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_multiply">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>287</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>287</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>287</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>170</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">NumberButtons</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button_minus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plus">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>287</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_equal">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>77</width>
      <height>77</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color: #111;</string>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="output_lcd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>291</width>
      <height>50</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>9</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="output_label_lcd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>57</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color: #111;</string>
    </property>
    <property name="text">
     <string>Infix</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_infix">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>291</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="NumberButtons"/>
 </buttongroups>
</ui>
